ENDEAVOURS-INDEX.md

```diff
@@ -0,0 +1,68 @@
+IPFS Endeavours Index
+=====================
+
+Our work around building and maintaining IPFS is broken up into numerous "endeavors", each of which has an assigned captain. This Roadmap indicates whether the endeavors are "Active", on the "Back Burner" or "Stale". You can use that to get a sense of where development efforts are currently focused. All of these endeavors are entirely open source. You can join in and contribute to any of the endeavors at any time.           
+
+## Active Endeavors
+
+"Active" endeavors are the endeavors for which we have allocated developer time this quarter.
+
+| Endeavor | Captain | Status |
+|---|---|---|
+js-ipfs | @diasdavid | Active |
+go-ipfs | @Kubuxu, @whyrusleeping | Active |
```

Comment:
I'm the captain of go-ipfs, there is only one captain per endeavor

Reply:
True. This was c&p from the previous table.